build/WFCORE-3769_WFLY-10305_galleon_build.adoc

```diff
@@ -0,0 +1,166 @@
+= Title
```

Comment:
I'm starting to wonder if the "Title" should just be blank in the template as it seems to not get changed a lot. I know I've forgotten several times :). Anyway this could probably use a title.